src/renderers/shared/fiber/ReactFiberScheduler.js

```diff
@@ -1005,6 +1034,30 @@ module.exports = function<T, P, I, TI, C, CX>(config : HostConfig<T, P, I, TI, C
     }
   }
 
+  function scheduleErrorRecovery(fiber : Fiber) {
+    scheduleUpdateAtPriority(fiber, TaskPriority);
+  }
+
+  function scheduleSetState(fiber : Fiber, partialState : any) {
```

Comment:
I had them outside at first but it was awkward because it required exposing access the current priority context, which seems leaky. Either the class module needs to be aware of the priority context or the scheduler needs to be aware of the update queue.